packages/expo-cli/src/commands/start.js

```diff
@@ -52,6 +52,10 @@ function parseStartOptions(projectDir: string, options: Object): Object {
     startOpts.maxWorkers = options.maxWorkers;
   }
 
+  if (options.offline) {
+    startOpts.offline = true;
```

Comment:
what if instead of passing this arg around we instead make a module that is responsible for storing the connection status? we could call it `ConnectionStatus` i guess and then here we could do `ConnectionStatus.setIsOffline(true)` and whenever we want to know the status from another module we can use `ConnectionStatus.isOffline()` or `ConnectionStatus.isOnline()`

Reply:
Yeah, definitely cleaner than passing the flag about 5 calls deep 

Sorry I'm just getting around to addressing this feedback now